demos/godd/snap/snapcraft.yaml

```diff
@@ -5,6 +5,7 @@ description:
  Written in go with support for device auto-detection via libgudev,
  you would need to use hw-assign to access devices.
 confinement: strict
+contact: mailto:michaelvogt+godd@imap.cc
```

Comment:
Does use of the mailto scheme suggest that there could be others? Like tel or http? If that's the case maybe this should be a list?